src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public class ColorSupplier {
+    public String getRandomColor()
```

Comment:
The method `getRandomColor` is declared but not implemented. You need to provide an implementation that returns a random color from the `Color` enum.